tests/lib-injection/dd-lib-python-init-test-django-unsupported-package/django_app.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from django.http import HttpResponse
+from django.urls import path
+
+
+filepath, extension = os.path.splitext(__file__)
+ROOT_URLCONF = os.path.basename(filepath)
+DEBUG = False
+SECRET_KEY = "fdsfdasfa"
+ALLOWED_HOSTS = ["*"]
+
+
+def index(request):
+    import ddtrace
+
+    if ddtrace.__version__ != "1.12.0":
```

Comment:
Why are we checking the ddtrace version and exiting if it's not 1.12 here?

Reply:
I copypasted that logic from previous lib-injection tests. Not really sure.